TUTORIAL_USE.md

```diff
@@ -0,0 +1,118 @@
+# Using the existing code generators
+
+Java::Geci provides several out-of-the-box code generators. 
+In this tutorial we'll use the generator `Accessor` as an example, 
+but the things explained in this tutorial apply to every other generator.
+
+You use Java::Geci by using the `Geci` class in tests.
+
+```java
+public class TestAccessor {
+
+    @Test
+    public void testAccessor() throws Exception {
+        Assertions.assertFalse(new Geci().source(maven()
+                        .module("javageci-examples").mainSource())
+                        .register(Accessor.builder().build()).generate(),
+                Geci.FAILED);
+    }
+}
+```
+This might seem like a lot at first, so let's break it down.
+
+`Geci` has a fluent interface.
+
+The method `source()` can be used to specify the directories where your
+source files are. If you have source files in different places you
+have to chain several `source()` invocations one after the other. Every
+single call to `source()` can specify several directories. These count
+as one single directory and the first that exists is used to discover
+the files.<br/>
+This means:
+* Call `source("foo").source("bar")` if you have sources in **both** foo **and** bar.
+* Call `source("foo", "bar")` if you have sources in **either** foo **or** bar.
+*maven().module("javageci-examples").mainSource()* is just "javageci-examples/src"
+
+The method `register()` can register one or more source code generators. 
+You can also chain `register()` calls and register your generators one-by-one. 
+It's up to you. Each registered generator will be invoked on the sources.
+ 
+Finally the method invocation `generate()` will do the work, read the
+source files and generate the code.
+
+Every generator has a mnemonic which identifies the generator. For example the Accessor
```

Comment:
No. Not every. Only the Java generators. Geci can start generators that have nothing to do with Java.

As a matter of fact I have a plan, and some features of Geci were designed so that the framework can handle some special features that are required by document generating/extracting generators.

Reply:
When this tutorial talks about "generators", I mean the core generators, provided by Java::Geci. Should probably clarify that in the beginning.